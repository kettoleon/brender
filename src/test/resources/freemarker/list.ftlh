Hi, your new products are: <#list communicationBody.triggeringProducts as prod>${prod.name} (${prod.price?string["#,##0.00;; groupingSeparator='.' decimalSeparator=','"]}â‚¬)<#if prod?index < communicationBody.triggeringProducts?size-2>, </#if><#if prod?index == communicationBody.triggeringProducts?size-2> and </#if></#list>.