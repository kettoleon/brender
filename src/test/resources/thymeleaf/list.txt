Hi, your new products are: [# th:each="prod : ${communicationBody.triggeringProducts}"][(${prod.name})] ([(${#numbers.formatDecimal(prod.price,1,'POINT',2,'COMMA')})]â‚¬)[# th:if="${prodStat.index} < ${prodStat.size}-2"], [/][# th:if="${prodStat.index} == ${prodStat.size}-2"] and [/][/].