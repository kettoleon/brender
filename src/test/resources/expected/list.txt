Hi, your new products are: product1 (4,50€), product2 (0,51€) and product3 (3.244,55€).